pkg/utils/kubelessutil.go

```diff
@@ -406,11 +407,14 @@ func populatePodSpec(funcObj *kubelessApi.Function, lr *langruntime.Langruntimes
 	}
 	if funcObj.Spec.Deps != "" && err != nil {
 		return fmt.Errorf("Unable to install dependencies for the runtime %s", funcObj.Spec.Runtime)
-	} else if funcObj.Spec.Deps != "" {
+	} else if funcObj.Spec.Deps != "" || funcObj.Spec.UseBundledDeps != "" {
```

Comment:
I would add the condition to a variable because the first `if` is also affected to this change:

```
hasDeps := funcObj.Spec.Deps != "" || strings.Contains(contentType, "deps")
if hasDeps != "" && err != nil {
  return fmt.Errorf("Unable to install dependencies for the runtime %s", funcObj.Spec.Runtime)
} else if hasDeps {
...
}
```